src/storages/types.ts

```diff
@@ -62,6 +64,18 @@ export interface ICustomStorageWrapper {
    * The promise rejects if the operation fails.
    */
   getByPrefix: (prefix: string) => Promise<string[]>
+  /**
+   * Returns the values of all given `keys`.
+   *
+   * @function getMany
+   * @param {string[]} keys list of keys to retrieve
+   * @returns {Promise<(string | null)[]>} A promise that resolves with the list of items associated with the specified list of `keys`. For every key that does not hold a string value or does not exist, null is returned.
```

Comment:
(Don't hate me): I know we are not using line Max length characters limitation, but let's try to make it shorter to make it more legible.

Reply:
hahaha. you are right. I will fix it, and we can add the rule but in another PR (it would imply hundreds of updates)